<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>ANNEHEIM Geoffrey</author>
 <class>MCServerMainWindow</class>
 <widget class="QMainWindow" name="MCServerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="MyQTabWidget" name="tabWidgetForms">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabServerLog">
       <attribute name="title">
        <string>ServerLog</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxServerConnection">
          <property name="title">
           <string>Server connection configuration</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="lblServerAddress">
                 <property name="text">
                  <string>Server address :</string>
                 </property>
                 <property name="buddy">
                  <cstring>textServerAddress</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblServerPort">
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Server port :</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                 <property name="buddy">
                  <cstring>textServerPort</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="textServerAddress">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>174</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string>127.0.0.1</string>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Normal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="textServerPort">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>174</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="inputMask">
                  <string>00000; </string>
                 </property>
                 <property name="text">
                  <string>8080</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="buttonServerListen">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxListOfClients">
          <property name="title">
           <string>Clients connected</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTreeWidget" name="treeWidgetClientsConnected">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <attribute name="headerDefaultSectionSize">
              <number>100</number>
             </attribute>
             <attribute name="headerMinimumSectionSize">
              <number>50</number>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP Address</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Port</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabClients">
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="doFilePreferences"/>
    <addaction name="separator"/>
    <addaction name="doFileExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuViewToolBars">
     <property name="title">
      <string>Toolbars</string>
     </property>
    </widget>
    <addaction name="menuViewToolBars"/>
    <addaction name="doViewStatusBar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="doHelpAbout"/>
    <addaction name="doHelpAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Main menu</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="doMainToolBarConnectDisconnect"/>
   <addaction name="separator"/>
   <addaction name="doMainToolBarServerLog"/>
   <addaction name="doMainToolBarClients"/>
   <addaction name="doMainToolBarPreferences"/>
   <addaction name="doMainToolBarExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="doMainToolBarConnectDisconnect">
   <property name="icon">
    <iconset>
     <activeoff>:/MainToolBar/UnconnectedIcon</activeoff>
     <activeon>:/MainToolBar/ConnectedIcon</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>ConnectDisconnect</string>
   </property>
   <property name="toolTip">
    <string>Connect / Disconnect</string>
   </property>
  </action>
  <action name="doMainToolBarServerLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/MainToolBar/ServerLogIcon</normaloff>:/MainToolBar/ServerLogIcon</iconset>
   </property>
   <property name="text">
    <string>ServerLog</string>
   </property>
   <property name="toolTip">
    <string>Show server log</string>
   </property>
   <property name="mcFormIndex" stdset="0">
    <number>0</number>
   </property>
  </action>
  <action name="doMainToolBarClients">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/MainToolBar/ClientsIcon</normaloff>:/MainToolBar/ClientsIcon</iconset>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
   <property name="toolTip">
    <string>View clients list</string>
   </property>
   <property name="mcFormIndex" stdset="0">
    <number>1</number>
   </property>
  </action>
  <action name="doMainToolBarExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/MainToolBar/ExitIcon</normaloff>:/MainToolBar/ExitIcon</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="doMainToolBarPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/MainToolBar/PreferencesIcon</normaloff>:/MainToolBar/PreferencesIcon</iconset>
   </property>
   <property name="text">
    <string>actionPreferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences dialog</string>
   </property>
  </action>
  <action name="doFileExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Menu/ExitIcon</normaloff>:/Menu/ExitIcon</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="doViewStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="doHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="doHelpAboutQt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Menu/AboutQtIcon</normaloff>:/Menu/AboutQtIcon</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="doFilePreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Menu/PreferencesIcon</normaloff>:/Menu/PreferencesIcon</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQTabWidget</class>
   <extends>QTabWidget</extends>
   <header>../Components/Widgets/MyQTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>doMainToolBarExit</sender>
   <signal>triggered()</signal>
   <receiver>MCServerMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MCServerMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doViewStatusBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
